sources/packages/api/src/types/config.ts

```diff
@@ -6,5 +6,5 @@ export interface IAuthConfig {
   url: string;
   realm: string;
   clientId: string;
-  issuerUrl: string;
+  OpenIdConfigurationUrl: string;
```

Comment:
why this property name starts with Capital letter?

Reply:
Because it is wrong :D
I am gonna change to the proper case.

Reply:
Fixed!